src/AST-Core/RBVariableNode.class.st

```diff
@@ -104,6 +102,7 @@ RBVariableNode >> hash [
 { #category : #initialization }
 RBVariableNode >> initialize [
 	super initialize.
+	variable := UnresolvedVariable instance.
```

Comment:
Little thought (not sure about it): 
We can use the #name of #variable as a name of the node and get rid of #name variable in the node. (hopefully you get this combination of words :))

Reply:
yes, understood... could be an idea to save a bit of memory